### PR TITLE
docs: Add documentation for feature DEPLOYMENT.md

### PR DESCRIPTION
**Pull request by OnahProsperity**
Improve the Deployment doc, the current documentation guide on deployment doesn't specify how to get fauset and deploy successfully:
image

### CODE REVIEW

**Review by CJ42 — COMMENTED** [review]
Nice and useful extra mention! I have added some comments.

Thank you @OnahProsperity! :)

File: DEPLOYMENT.md [hunk 1]
```diff
@@ -22,7 +22,11 @@ All the deployment scripts for `base` contracts initialize the contract after de
 
 &nbsp;
 
-2. run the command using one of the available `--tags`
+2. Visit [LUKSO l14 Fauset](http://faucet.l14.lukso.network/), for fauset.
```
[the remark this answers is not in the document]
Comment (OnahProsperity) [reply DEPLOYMENT.md]: Thanks for drawing my attention to it once again, the correction has now been updated as suggested.
Thanks.

File: DEPLOYMENT.md [hunk 2]
```diff
@@ -1,7 +1,8 @@
 
 # Deployment
 
-You can find a deployment utility with hardhat to easily deploy the smart contracts locally or on our L14 test network.
+You can find a deployment utility with hardhat to easily deploy the smart contracts locally or on our L14 test network,
+if you don't have some LYX test token visit [LUKSO l14 Fauset](http://faucet.l14.lukso.network/), for fauset.
```
Comment (CJ42) [inline DEPLOYMENT.md:5]: @OnahProsperity sorry for the misunderstanding. Maybe we did not state it the correction properly 🙂 

Could you replace the words as follow please:

- `Fauset` -> `Faucet`
- `fauset` -> `faucet`

with a **`c`**, not a **`s`**.

**Review by CJ42 — COMMENTED** [review]
see comments

File: DEPLOYMENT.md (hunk 2 above)
Comment (OnahProsperity) [reply DEPLOYMENT.md]: Thanks.
Updated.

**Review by YamenMerhi — APPROVED** [review]

**Comment by CJ42** [comment]
Thank you @OnahProsperity !

**Pull request merged**